gbpservice/nfp/core/log.py

```diff
@@ -0,0 +1,123 @@
+from oslo_log import log as oslo_logging
+
+import inspect
```

Comment:
Is this package available with devstack installation. If not, we need to think of adding this to dependency list.